demo/node_1.log

```diff
@@ -1,3 +1,3 @@
-1 root cause - fake
-2 root cause
-3 visible effect
+2015-12-03 12:08:07 root cause - fake
```

Comment:
this was an epoch time test (number of seconds since 1970 as an integer). You can change the datetime format only if some other test uses epoch time (create one if necessary)